src/biokbase/auth.py

```diff
@@ -29,59 +53,43 @@ def validate_token():
         return False
 
 
-def set_environ_token(token):
+def set_environ_token(token: str) -> None:
     """
     Sets a login token in the local environment variable.
     """
     kbase_env.auth_token = token
 
 
-def get_auth_token():
+def get_auth_token() -> Optional[str]:
     """
     Returns the current login token being used, or None if one isn't set.
     """
     return kbase_env.auth_token
 
 
-def get_user_info(token):
-    """
-    This uses the given token to query the authentication service for information
-    about the user who created the token.
-    """
+def get_token_info(token: str) -> TokenInfo:
     headers = {"Authorization": token}
     r = requests.get(token_api_url + endpt_token, headers=headers)
     if r.status_code != requests.codes.ok:
         r.raise_for_status()
```

Comment:
I know this is existing code, but you could change this to remove the `if r.status_code != requests.codes.ok` as `r.raise_for_status()` will throw an error if the status code is not OK.

Reply:
Done. 👍